.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,47 @@
+name: "CodeQL"
```

Comment:
Why this new service and why only on this repo?

Reply:
I can drop it into the other apps. It wasn't part of the Travis CI and is something that github actions can easily drop in if you start with actions via the website rather than code.